### PR TITLE
Enable pop sql logging in development

### PR DESCRIPTION
**Pull request by gidjin**
## Description

To aid developers in identifying SQL issues during development it was suggested to enable SQL logging by default in the local environment.

## Setup

Start the server and interact with the app in a way that triggers a SQL query. For example, `Local Sign In | ` and you should see some queries in the server log. 

```sh
make server_run
```

Test that it can be disabled and that no SQL queries show up in the log.

```sh
DB_DEBUG=0 make server_run
```

## Code Review Verification Steps

* [ ] Code follows the guidelines for [Logging](https://github.com/transcom/mymove/tree/master/docs/backend.md#logging)
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [#168392051](https://www.pivotaltracker.com/story/show/168392051) for this change
* [Slack Discussion](https://defensedigitalservice.slack.com/archives/G8HJSJE67/p1568236072205800) explains more about the approach used.

### CODE REVIEW

File: docs/database.md [hunk 1]
```diff
@@ -2,3 +2,15 @@
 
 * [How To Backup and Restore the Development Database](how-to/backup-and-restore-dev-database.md#how-to-backup-and-restore-the-development-database)
 * [How To Migrate the Database](how-to/migrate-the-database.md#how-to-migrate-the-database)
+
+## POP SQL logging on by default in development
+
+POP is an ORM which helps ease communication with the database by providing boiler plate code. However, this can obscure the actual SQL that is being executed without a in depth knowledge of the ORM. By enabling SQL logging in development a developer can see the queries being executed by POP as they are being executed to hopefully help developers to catch issues in the setup of queries with POP.
```
Comment (reggieriser) [inline docs/database.md]: A few nits:
- `Pop`, not `POP` -- and maybe link to the github page for the project
- `boilerplate`, not `boiler plate` -- although I think I'd classify Pop more of a database API abstraction than boilerplate code.
- `a in depth` -> `an in-depth`
- "queries being executed by POP as they are being executed" -- seems redundant

Should we also put a couple of examples of the types of things we could discover via the SQL logging (with supporting links as appropriate)?  Thinking about:
- excessive queries (e.g., `n+1` problem)
- excessive joins (e.g., open-ended `Eager` call)

Comment (gidjin) [reply docs/database.md]: I like the idea of adding examples of problems we might see. I'll see if I can find some good supporting links, at least about those two problems. Not sure what other problems would be good to cover. I'll also update the doc according to the other feedback as well.

**Comment by jim** [comment]
I am a big fan of this change.

**Comment by gidjin** [comment]
I posted in slack to announce this, probably should also announce it at our next demo too.

**Pull request merged**